warehouse/utils/wsgi.py

```diff
@@ -104,8 +132,31 @@ def _remote_addr_hashed(request: Request) -> str:
     return request.environ.get("REMOTE_ADDR_HASHED", "")
 
 
+def _ip_address(request):
+    """Return the IpAddress object for the remote address from the environment."""
+    try:
+        ip_address = (
+            request.db.query(IpAddress)
+            .filter(IpAddress.ip_address == request.remote_addr)
```

Comment:
We could write a simpler query:

```suggestion
            .filter_by(ip_address=request.remote_addr)
```

Both generate the same SQL query under the hood, just a matter of what looks more pleasing.

Same comment applies to queries in the test code.

Reply:
https://github.com/pypi/warehouse/pull/13594/commits/d9357aaf68aed31877e9c717849dc72334599860